c1-mauer-rose

kfahn22 opened the pull request:
Adding c1-mauer-rose to coding challenges.  I have the rest  of coding in the cabana as well, but will create pull requests in order.

Note:  I do not have the index.png.

shiffman commented on content/videos/challenges/c1-mauer-rose/index.json:
```diff
@@ -0,0 +1,71 @@
+{
+    "title": "Maurer Rose",
+    "description": "I code a visualization of the Maurer Rose pattern.",
```
```suggestion
    "description": "Coding in the Cabana is a series where I attempt challenges from my garden cabana in Brooklyn, NY. In this episode, I code a visualization of the Maurer Rose patter in JavaScript (p5.js).",
```

shiffman commented on content/videos/challenges/c1-mauer-rose/index.json:
```diff
@@ -0,0 +1,71 @@
+{
+    "title": "Maurer Rose",
+    "description": "I code a visualization of the Maurer Rose pattern.",
+    "videoNumber": "c1",
```
```suggestion
    "videoNumber": "1 - Coding in the Cabana",
```

shiffman commented on content/videos/challenges/c1-mauer-rose/index.json:
```diff
@@ -0,0 +1,71 @@
+    ],
+    "codeExamples": [
+      {
+        "title": "Mauer Rose",
```
```suggestion
        "title": "Maurer Rose",
```

shiffman commented on content/videos/challenges/c1-mauer-rose/index.json:
```diff
@@ -0,0 +1,71 @@
+    "codeExamples": [
+      {
+        "title": "Mauer Rose",
+        "description": "Implementation of mauer rose",
```
```suggestion
        "description": "Visualization of the Maurer rose pattern",
```

shiffman commented on content/videos/challenges/c1-mauer-rose/index.json:
```diff
@@ -0,0 +1,71 @@
+        "image": "rose.png",
+        "urls": {
+          "p5": "https://editor.p5js.org/codingtrain/sketches/qa7RiptE9",
+          "processing": "https://github.com/CodingTrain/website-archive/tree/main/challenges/coding-in-the-cabana/001_maurer_rose/Processing"
```
```suggestion
          "processing": "https://github.com/CodingTrain/Coding-in-the-Cabana/tree/main/001-Maurer-Rose/"
```

shiffman reviewed (commented):
Thank you for this! Made some minor adjustments, note the addition of a new Repo for Coding in the Cabana example code!

https://github.com/CodingTrain/Coding-in-the-Cabana/

kfahn22 commented:
I have edited the remaining challenges in Coding in the Cabana.  (I wasn't sure what was changed in the third commit, though.)  Here are the edits I made.
-updated the video number
-updated the description of the video
-updated the description for example code
-updated the link to repo
Please let me know if there is anything else to change.

shiffman commented:
All sounds good! I have to move the other code examples over too! Let me merge this one and we can tackle the others in separate pull requests!
